src/ui/Roster.ts

```diff
@@ -53,6 +57,7 @@ export default class Roster {
       $('.jsxc-js-notice-menu .jsxc-menu__button').text('');
 
       this.contactList = this.element.find('.jsxc-contact-list');
+     this.groupList = this.element.find('.jsxc-group-list');
```

Comment:
We should run `yarn fix-typescript-format` to fix the code style.

Reply:
inserted space..